### PR TITLE
feat(README.md): add my name to README as attendee

### PR DESCRIPTION
**Pull request by VeeIsForVanana**
This PR adds my name (in the FN MI LN scheme) under the attendees section of this document in regular format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ This repository serves as a check-in for attendees of the **Git Workshop for Mod
 ---
 ## Attendees
 
+Victor Edwin E Reyes
```
Comment (BastiDood) [inline README.md:8]: Just curious, why not add the period after the middle initial?
```suggestion
* Victor Edwin E. Reyes
```

**Review by BastiDood — CHANGES_REQUESTED** [review]

**Comment by BastiDood** [comment]
Almost there! However, now it's a bit difficult to merge this back to `main` because it now conflicts with the latest changes. Normally, all we have to do is:

1. Sync the `main` branch of the fork.
2. Switch to the feature branch.
3. Run `git merge main` to incorporate the latest changes from `main` into our feature branch.
4. Resolve the merge conflicts manually.
5. Create a new commit for the merge resolution.
6. Push the commits to the upstream feature branch.

However, since you pushed your commits directly into the `main` branch of your fork, then the **Sync Fork** button won't work. We can't merge the upstream `main` into your fork's `main` because they have already diverged.

Unfortunately, the easiest way to solve this (for now) is to just delete your fork and start from scratch. This does mean I'll have to close this PR, but that's okay! This is a learning experience in open-source contribution. Now you know that feature branches are important.

File: README.md (hunk 1 above)
Comment (VeeIsForVanana) [reply README.md:8]: It's an old tradition I picked up from my days in the corps. There, if I remember correctly, you only put periods after middle initials if the person was dead. In fact even abbreviations in ranks and titles lack periods (e.g. C2Lt Victor Edwin E Reyes 3Cl). 

Of course, I've resigned now, but I still just never put periods after any living person's middle initials out of habit. That and it's just a fun check if the person parsing is paying attention (as in this case!). Though, I'd gladly put one down in line with the requirements of any regex parser or other formatting requirement.

**Comment by VeeIsForVanana** [comment]
> Almost there! However, now it's a bit difficult to merge this back to `main` because it now conflicts with the latest changes. Normally, all we have to do is:
> 
> 1. Sync the `main` branch of the fork.
> 2. Switch to the feature branch.
> 3. Run `git merge main` to incorporate the latest changes from `main` into our feature branch.
> 4. Resolve the merge conflicts manually.
> 5. Create a new commit for the merge resolution.
> 6. Push the commits to the upstream feature branch.
> 
> However, since you pushed your commits directly into the `main` branch of your fork, then the **Sync Fork** button won't work. We can't merge the upstream `main` into your fork's `main` because they have already diverged.
> 
> Unfortunately, the easiest way to solve this (for now) is to just delete your fork and start from scratch. This does mean I'll have to close this PR, but that's okay! This is a learning experience in open-source contribution. Now you know that feature branches are important.

Copy that, I'll make a new PR via a new fork and feature branch. Thanks!

**Pull request closed without merging**